src/driver/safe.rs

```diff
@@ -1230,7 +1314,9 @@ extern \"C\" __global__ void sin_kernel(float *out, const float *inp, size_t num
 
         for i in 0..5 {
             let a_sub = a_dev.try_slice(i * 2..).unwrap();
+            assert_eq!(a_sub.len, 10 - 2 * i);
             let mut b_sub = b_dev.try_slice_mut(i * 2..).unwrap();
+            assert_eq!(b_sub.len, 10 - 2 * i);
```

Comment:
Want to adjust this test to use `i*2..i*2+2`?

Reply:
hmm, this is actually an existing test where i just added to test the length (while this should probably be seperated ...). But what exactly do you mean? the slicing with i*2..i*2+2; if yes, why?

Reply:
Mainly want something to test that the length of CudaView is properly used. I think this can be done in #56